pkg/agent/secondarynetwork/init.go

```diff
@@ -48,13 +50,38 @@ func Initialize(
 	nodeName string,
 	podUpdateSubscriber channel.Subscriber,
 	stopCh <-chan struct{},
-	config *agentconfig.SecondaryNetworkConfig, ovsdb *ovsdb.OVSDB) error {
+	secNetconfig *agentconfig.SecondaryNetworkConfig, ovsdb *ovsdb.OVSDB) error {
 
-	ovsBridgeClient, err := createOVSBridge(config.OVSBridges, ovsdb)
+	ovsBridgeClient, err := createOVSBridge(secNetconfig.OVSBridges, ovsdb)
 	if err != nil {
 		return err
 	}
 
+	phyInterfaces := make([]string, len(secNetconfig.OVSBridges[0].PhysicalInterfaces))
```

Comment:
So secNetConfig is passed as a pointer, and I'm updating the name of the interface `eth0` -> `eth0~` just before calling `connectPhyInterfacesToOVSBridge(phyInterfaces)`. 
`phyInterfaces := secNetconfig.OVSBridges[0].PhysicalInterfaces` this would create a shallow copy and renaming interface name in `phyInterfaces` would mutate the original config struct right ?